src/api/rest/server/server.go

```diff
@@ -169,8 +171,13 @@ func RunServer(port string) {
 	e.POST("/tumblebug/mcisDynamicCheckRequest", rest_mcis.RestPostMcisDynamicCheckRequest)
 
 	g.POST("/:nsId/mcisDynamic", rest_mcis.RestPostMcisDynamic)
-	g.GET("/:nsId/mcis/:mcisId", rest_mcis.RestGetMcis)
-	g.GET("/:nsId/mcis", rest_mcis.RestGetAllMcis)
+
+	//g.GET("/:nsId/mcis/:mcisId", rest_mcis.RestGetMcis, middleware.TimeoutWithConfig(middleware.TimeoutConfig{Timeout: 20 * time.Second}), middleware.RateLimiter(middleware.NewRateLimiterMemoryStore(1)))
+	//g.GET("/:nsId/mcis", rest_mcis.RestGetAllMcis, middleware.TimeoutWithConfig(middleware.TimeoutConfig{Timeout: 20 * time.Second}), middleware.RateLimiter(middleware.NewRateLimiterMemoryStore(1)))
+	// path specific timeout and ratelimit
+	g.GET("/:nsId/mcis/:mcisId", rest_mcis.RestGetMcis, middleware.TimeoutWithConfig(middleware.TimeoutConfig{Timeout: 60 * time.Second}), middleware.RateLimiter(middleware.NewRateLimiterMemoryStore(1)))
+	g.GET("/:nsId/mcis", rest_mcis.RestGetAllMcis, middleware.TimeoutWithConfig(middleware.TimeoutConfig{Timeout: 60 * time.Second}), middleware.RateLimiter(middleware.NewRateLimiterMemoryStore(1)))
```

Comment:
nit: L175-L176 삭제 + 가독성 및 코드 유지보수 편의성 관련 입니다.. 😊
(실제로 빌드해 보지는 않아서.. Syntax error 날 수도 있겠네요..
아니면 gofmt 가 자동으로 원복시키거나.. 😅)

```suggestion
	// path specific timeout and ratelimit

	g.GET("/:nsId/mcis/:mcisId", rest_mcis.RestGetMcis, 
	middleware.TimeoutWithConfig(middleware.TimeoutConfig{Timeout: 60 * time.Second}), 
	middleware.RateLimiter(middleware.NewRateLimiterMemoryStore(1)))

	g.GET("/:nsId/mcis", rest_mcis.RestGetAllMcis, 
	middleware.TimeoutWithConfig(middleware.TimeoutConfig{Timeout: 60 * time.Second}), 
	middleware.RateLimiter(middleware.NewRateLimiterMemoryStore(1)))
```